endpoints/OAI/router.py

```diff
@@ -166,3 +168,28 @@ async def embeddings(request: Request, data: EmbeddingsRequest) -> EmbeddingsRes
     )
 
     return response
+
```

Comment:
The route should be in endpoints/core since those are propagated across all API servers.